src/cache/index.ts

```diff
@@ -15,4 +15,9 @@ export const cache = new InMemoryCache({
     },
 });
 
-export const dashboardItemsVar = makeVar<DashboardItemInfo[]>([]);
+const initValue = [];
+const storage = window.localStorage.getItem('dashboard');
+if (storage) {
+    initValue.push(...JSON.parse(storage));
```

Comment:
Could you `let initValue: Type = []` and simply assign instead? Would be a bit easier to understand.